source/extensions/common/wasm/wasm.cc

```diff
@@ -312,8 +313,9 @@ WasmEvent toWasmEvent(const std::shared_ptr<WasmHandleBase>& wasm) {
   NOT_IMPLEMENTED_GCOVR_EXCL_LINE;
 }
 
-static bool createWasmInternal(const VmConfig& vm_config, const PluginSharedPtr& plugin,
-                               const Stats::ScopeSharedPtr& scope,
+static bool createWasmInternal(const VmConfig& vm_config,
+                               const CapabilityRestrictionConfig& cr_config,
```

Comment:
Nit: `s/cr_config/capabilities_config`, otherwie variable isn't self-describing.

Reply:
Fixed, changed to capabiility_restriction_config for consistency with type name and proto. Can change to capabilities_config if too long.